integration-tests/rest-tests-jaxrs/src/test/groovy/org/hawkular/metrics/rest/TagsITest.groovy

```diff
@@ -297,17 +297,27 @@ class TagsITest extends RESTTest {
 
       response = hawkularMetrics.get(path: it.path + "/tags/a1:*,d1:B%3AA",
           headers: [(tenantHeaderName): tenantId])
-      assertEquals(200, response.status)
 
-      assertEquals([
-                        a1: ['a', 'A/B'],
-                        d1: ['B:A']
-                    ], response.data)
+      def genericResponse = hawkularMetrics.get(path: "metrics/tags/a1:*,d1:B%3AA",
+          query: [type: it.type], headers: [(tenantHeaderName): tenantId])
+
+      def untypedResponse = hawkularMetrics.get(path: "metrics/tags/a1:*,d1:B%3AA", headers: [(tenantHeaderName):
+                                                                                                  tenantId])
+
+      [response, genericResponse, untypedResponse].each { lresponse ->
+        assertEquals(200, lresponse.status)
+        assertEquals([
+            a1: ['a', 'A/B'],
+            d1: ['B:A']
+        ], lresponse.data)
```

Comment:
If the result is always the same, it means the user get the metric name but not the type when querying without the type info, correct? Can you elaborate on the use case?